docs/standard/serialization/system-text-json-converters-how-to.md

```diff
@@ -0,0 +1,755 @@
+---
+title: "How to write custom converters for JSON serialization - .NET"
+author: tdykstra
+ms.author: tdykstra
+ms.date: "10/16/2019"
+helpviewer_keywords: 
+  - "JSON serialization"
+  - "serializing objects"
+  - "serialization"
+  - "objects, serializing"
+  - "converters"
+---
+
+# How to write custom converters for JSON serialization in .NET
+
+This article shows how to create custom converters for the JSON serialization classes that are provided in the <xref:System.Text.Json> namespace. For an introduction to `System.Text.Json`, see [How to serialize and deserialize JSON in .NET](system-text-json-how-to.md).
+
+A *converter* is a class that converts an object or a value to and from JSON. The `System.Text.Json` namespace has built-in converters for most primitive types that map to JavaScript primitives. You can write custom converters:
+
+* To override the default behavior of a built-in converter. For example, you might want `DateTime` values to be represented by mm/dd/yyyy format instead of the default  ISO 8601-1:2019 format.
+* To support a custom data type. For example, a `PhoneNumber` struct.
```

Comment:
I believe we generally refer to `struct ` as value types in the documentation, no?